### PR TITLE
New - Testnet Quick Start Guide

### PR DESCRIPTION
**Pull request by bobgt**
These are the files for EOSIO Testnet Quick Start Guide. 

Please review the content and provide suggestions/feedback of any nature. Google doc for this guide is here:
https://docs.google.com/document/d/17y7J64Fl5YoHp-ujSmBnEtcDG83iqw4uNbAtyCees5c/edit# 

**Note**: Screenshots for the below topics are to be added (WIP):
1. Buy and Stake System Resources
2. View Account Details
3. Smart Contract Deployment
4. Push Actions
5. View Transactions

Keeping this a draft PR for now as the screenshots upload is WIP. In the meanwhile, it would be great to have any improvement suggestions on the core content as we are aiming to merge this by tomorrow EOD to coincide with the Tic Tac Toe publish.

### CODE REVIEW

**Comment by halsaphi** [comment]
Change images dir name to resources - or vice versa if this requires discussion.
Multi-index Table - title for the "View Actions Data in Multi-index Table" should this be "View Actions Data in Multi Index Table"

**Comment by bobgt** [comment]
> Change images dir name to resources - or vice versa if this requires discussion.
> Multi-index Table - title for the "View Actions Data in Multi-index Table" should this be "View Actions Data in Multi Index Table"

1. As we checked, the existing directory names is images so let's continue with `images`
2. As discussed, let's use **Multi-Index Table**.

File: docs/08_testnet-quick-start-quide/index.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+**Complete the following steps to push actions on the Testnet:**
+
+1. Navigate to the **Push Action** page from the top navigation menu.
+
+   ![SM Upload](./images/push-action-1.png)
+2. Enter the account name on which the smart contract is deployed and select the action type in the **Action Type** drop-down menu.
+
+   ![SM Upload](./images/push-action-2.png)
+3. Enter the JSON payload containing the data or parameters implemented in the source files as shown in the example below.
```
[the remark this answers is not in the document]
Comment (halsaphi) [reply docs/08_testnet-quick-start-quide/index.md:195]: one comment here should it be json or JSON - I'm using json (lower case) so one of us needs to change it for consistency - don't mind which

File: docs/08_testnet-quick-start-quide/index.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
+# **Abstract**
```
[the remark this answers is not in the document]
Comment (bobgt) [reply docs/08_testnet-quick-start-quide/index.md:1]: I have seen Abstract being used in developer guides but Overview sounds good. I will change it.

File: docs/08_testnet-quick-start-quide/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bobgt) [reply docs/08_testnet-quick-start-quide/index.md:195]: JSON is represented in caps/uppercase so let's use the caps version.

**Comment by bobgt** [comment]
Thanks, Luis for your helpful suggestions. I've applied them.

**Comment by bobgt** [comment]
@iamveritas: is `release/2.0.x` the correct branch to merge this?

**Comment by lparisc** [comment]
> @iamveritas: is `release/2.0.x` the correct branch to merge this?

Since the EOSIO Testnet has been around even before EOSIO 2.0, I'd we should publish it right away. It should also go to `develop` of course.

**Review by halsaphi — APPROVED** [review]

**Pull request merged**